Dockerfile

```diff
@@ -205,6 +205,10 @@ RUN mkdir -p /usr/src/awsclizip "${LABKEY_HOME}/awsclibin" "${LABKEY_HOME}/aws-c
     && rm -rf /usr/src/awsclizip \
     && chown -R labkey:labkey "${LABKEY_HOME}/awsclibin" "${LABKEY_HOME}/aws-cli"
 
+# install datadog tracing agent
+RUN mkdir -p datadog \
+    && wget -q -O datadog/dd-java-agent.jar https://dtdg.co/latest-java-tracer
```

Comment:
Do we want to pin this to a particular version or always grab that latest? I know we generally want to stay current, but sometimes it's painful to not have more explicit control.

Reply:
We've been allowing cloud deployments to always pull the latest version of the main datadog agent and tracer for most of a year without an issue. I feel like their dev/release processes are very robust and would rather take that risk than have one more thing to version bump. If it broke it would probably fail the teamcity build anyway (if we enable all the datadog features in the build job, which seems prudent).

Reply:
OK!